tests/TestCase/Http/ControllerFactoryTest.php

```diff
@@ -263,4 +263,23 @@ public function testAbsoluteReferenceFailure()
         ]);
         $this->factory->create($request, $this->response);
     }
+
+    /**
+     * Test building controller name passing no controller name
```

Comment:
Sentence is somewhat confusing, may be changing to "name when passing" helps )If I understood correctly).